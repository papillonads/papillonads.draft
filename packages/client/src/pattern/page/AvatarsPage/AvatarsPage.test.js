import React from 'react'
import avatarsPageDefault from './AvatarsPage.render'

jest.mock('react-apollo', () => {
  return {
    Query: () => <div />,
  }
})

describe('<AvatarsPage />', () => {
  describe('Snaphot', () => {
    test('must match avatarsPageDefault', () => {
      expect(global.renderToJSON(avatarsPageDefault)).toMatchSnapshot()
    })
  })
})
