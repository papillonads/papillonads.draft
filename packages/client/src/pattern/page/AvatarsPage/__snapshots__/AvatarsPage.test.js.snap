// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AvatarsPage /> Snaphot must match avatarsPageDefault 1`] = `<div />`;
